src/audio/midi/mod.rs

```diff
@@ -30,19 +34,19 @@ pub fn port_names() -> Result<Vec<String>> {
 }
 
 /// Get the last value for a `ControlFunction` from a list of messages.
-pub fn latest_control_value(
-    slider: MidiSlider,
-    messages: &[MidiMessage<'static>],
-) -> Option<ControlValue> {
-    messages.iter().rev().find_map(|message| match message {
-        MidiMessage::ControlChange(ch, function, value)
-            if ch == &*slider.channel && function == &*slider.control_change =>
-        {
-            Some(*value)
-        }
-
-        _ => None,
-    })
+pub fn latest_control_value(slider: MidiSlider, messages: &[Message]) -> Option<ControlValue> {
+    messages
+        .iter()
+        .rev()
+        .find_map(|message| match message.message {
+            MidiMessage::ControlChange(ch, function, value)
+                if ch == *slider.channel && function == *slider.control_change =>
+            {
+                Some(value)
```

Comment:
It's because of the changes to the definition of `MidiSlider` and the channel and control change types. Before, I had defined my own types which could be "derefed" (https://doc.rust-lang.org/book/ch15-02-deref.html) into the underlying types from the MIDI library, but now I'm just using the library types directly. The way it was working before was overly complicated to try and work around a missing feature in serde which it turns out isn't actually missing 😄